mathics/builtin/assignment.py

```diff
@@ -625,6 +627,11 @@ class Definition(Builtin):
     >> Definition[a]
      = a = 2
 
+    >> a = 2;
+    >> ? a
+     = a = 2
+
+
```

Comment:
I use the version 9, and Hold[?a] also returns Hold[Information[a, LongForm -> False]]
,  but the help says something different: https://reference.wolfram.com/language/ref/Definition.html